src/main/java/io/dataspaceconnector/common/net/HttpService.java

```diff
@@ -161,10 +166,15 @@ public Response get(final URL target, final HttpArgs args) throws IOException {
         Utils.requireNonNull(args, ErrorMessage.HTTP_ARGS_NULL);
 
         final var urlBuilder = createUrlBuilder(target);
+        HttpUrl targetUrl = toUrl(target);
```

Comment:
Please follow the coding standard of the project and use `final var`:
`final var targetUrl = ...` ?